index.html

```diff
@@ -53,5 +53,255 @@ <h2>I'm a Software Developer</h2>
+              industry's standard
+          </p>
+          <ul>
+              <li>
+                  html
+              </li>
+              <li>
+                  bootstrap
+              </li>
+              <li>
+                  Ruby
+              </li>
+          </ul>
+          <button type="button">See project</button>
+      </div>
+      <div id="normal">
+          <img src="./img/work.png" alt="Image">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the
+              industry's standard
+          </p>
+          <ul>
+              <li>
+                  html
+              </li>
+              <li>
+                  bootstrap
+              </li>
+              <li>
+                  Ruby
+              </li>
+          </ul>
+          <button type="button">See project</button>
+      </div>
+      <div id="normal">
+          <img src="./img/work.png" alt="Image">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the
+              industry's standard
+          </p>
+          <ul>
+              <li>
+                  html
+              </li>
+              <li>
+                  bootstrap
+              </li>
+              <li>
+                  Ruby
+              </li>
+          </ul>
+          <button type="button">See project</button>
+      </div>
+      <div id="normal">
+          <img src="./img/work.png" alt="Image">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the
+              industry's standard
+          </p>
+          <ul>
+              <li>
+                  html
+              </li>
+              <li>
+                  bootstrap
+              </li>
+              <li>
+                  Ruby
+              </li>
+          </ul>
+          <button type="button">See project</button>
+      </div>
+      <div id="normal">
+          <img src="./img/work.png" alt="Image">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. has been the
+              industry's standard
+          </p>
+          <ul>
+              <li>
+                  html
+              </li>
+              <li>
+                  bootstrap
+              </li>
+              <li>
+                  Ruby
+              </li>
+          </ul>
+          <button type="button">See project</button>
+      </div>
+  </section>
+
+  <section class="about">
+      <div class="brief">
+          <h1>About me</h1>
+          <p>Hello I’m a software developer!
```

Comment:
- [ ] Good job adding the about section to your portfolio  👏 , but I think it could be better to reduce the space in `red` , As it will make your work sticks to the Figma design. 
- [ ] For more information about he Figma design see [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

 ||||
 | :---: | :---: | :---: |
 | ![Capture 1](https://user-images.githubusercontent.com/80458199/227169870-843e9979-12ed-4554-b429-52e7fe29601d.JPG) | ![Capture 2](https://user-images.githubusercontent.com/80458199/227172152-6f03b901-61da-4f44-b0b1-d5b3989b9326.JPG) | ![Capture](https://user-images.githubusercontent.com/80458199/227172202-11ac2f0b-c79d-419c-bd60-30bb0ac824fd.JPG) |